docs/src/pages/customization/transitions/transitions.md

```diff
@@ -0,0 +1,33 @@
+# Transitions
+
+<p class="description">Use the <code>theme.transitions.create()</code> helper to create consistent transitions on the elements of your UI.</p>
+
+{{"demo": "pages/customization/transitions/TransitionHover.js"}}
```

Comment:
I don't think that the inline preview is of any value. We can disable it to reduce noise.

```suggestion
{{"demo": "pages/customization/transitions/TransitionHover.js", "defaultCodeOpen": false}}
```

Reply:
Or we could do the opposite, open it completely
```suggestion
{{"demo": "pages/customization/transitions/TransitionHover.js", "defaultCodeOpen": true}}
```

Reply:
Let me change the example to see how long it would be and I will check what will look better